SystemManager/Inc/SM.hpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * SM.hpp
+ *
+ * System Manager Header
+ *
+ * Created on: Nov 7, 2022
+ * Author(s): Gordon Fountain
+ */
+
+#ifndef ZPSW3_SM_HPP
+#define ZPSW3_SM_HPP
+
+#include <stdint.h>
+// #include "LOS_Link.hpp"
+// #include "LOS_Actuators.hpp"
+// #include "AM_StateManager.hpp"
+
+class SystemState;
+
+namespace SM {
+    // Enumerates the current drone status
+    enum _System_Manager_Drone_Operation_Mode{BOOT, DISARMED, GROUND_OPS, FLIGHT, FATAL_FAILURE = -1};
+
+    enum _System_Manager_Cycle_Status {COMPLETED_CYCLE = 0, IN_CYCLE, FAILURE_MODE};
+}
+
+class SystemManager {
+public:
+    SystemManager();
+    void execute();
+private:
+    SystemState* currentState;
+    SM::_System_Manager_Cycle_Status status;
+    SystemManager();
+
+    // TODO Bulk message from telemeetry stored here
+
+    // TODO Responses from PM/AM stored here to be merged and sent to Telemetry
+    inputs_to_AM_t* AM_Waypoints;
```

Comment:
Not important not but I think this should be renamed from AM_Waypoints to something else to avoid confusion? Since PM is giving AM a vector, not a waypoint